spago.yaml

```diff
@@ -17,13 +17,7 @@ workspace:
     cmd: purescm
     args:
       - "build"
-  extra_packages:
-    prelude:
-      git: https://github.com/purescm/purescript-prelude.git
-      ref: ab7dfc91e44096254caae017c2fce12d50ae114a
-    foldable-traversable:
-      git: https://github.com/purescm/purescript-foldable-traversable.git
-      ref: 218fb8f79f5eb85dfd9ede31d5767a71f2eeaa77
   package_set:
     url: https://raw.githubusercontent.com/purescm/purescm/master/package-sets/1.0.0.json
```

Comment:
You should just replace `master` with the ref, as this is just a url and not a url to a git repo.

Reply:
😅